stacks/container-apps/web-application.yml

```diff
@@ -230,6 +235,76 @@ Resources:
     Type: "AWS::Logs::LogGroup"
     Properties:
       RetentionInDays: 14
+  CourierWebTaskIamRole:
```

Comment:
Oof, parameterizing roles seems like a pain.  Maybe we'll just store these in another stack and import them?  (Just explicitly define them for every app).